CoreScriptsRoot/CoreScripts/Topbar.lua

```diff
@@ -1840,7 +1846,15 @@ if gameOptions and not isTenFootInterface then
 end
 
 local function topBarEnabledChanged()
-	topbarEnabledChangedEvent:Fire(topbarEnabled)
+	if InputService.VREnabled then
+		lookMenuEnabled = topbarEnabled
+		topbarEnabled = false
```

Comment:
Seems a little dicey to me to change this property away from what the dev set it.